app/controllers/api/reviews_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Api::ReviewsController < ApiController
+
+  def show
+    @review = Review.find(params[:id])
```

Comment:
this is *waaaaaaay* too big to be in a controller action. I think you could extract this into a PORO, or at the very least, private methods in this controller.